core/src/test-integration/java/io/neow3j/protocol/Neow3jIntegrationTest.java

```diff
@@ -103,6 +137,17 @@ private void waitUntil(Callable<?> callable, Matcher matcher) {
             };
         }
 
+        private void makeTransaction() throws IOException {
```

Comment:
Which transaction? 😄 

```suggestion
        private void performSendToAddressTransaction() throws IOException {
```